MPF.Core/Modules/CleanRip/Parameters.cs

```diff
@@ -182,6 +184,30 @@ public override List<string> GetLogFilePaths(string basePath)
                         sha1 = line.Substring(7);
                 }
 
+                // Ensure all checksums were found in log
+                if (crc == string.Empty || md5 == string.Empty || sha1 == string.Empty)
+                {
+                    if (HashTool.GetStandardHashes(iso, out long isoSize, out string? isoCRC, out string? isoMD5, out string? isoSHA1))
+                    {
+                        if (crc != string.Empty && crc != isoCRC)
+                        {
+                            // Warning
+                        }
+                        if (md5 != string.Empty && md5 != isoMD5)
+                        {
+                            // Warning
+                        }
+                        if (sha1 != string.Empty && sha1 != isoSHA1)
+                        {
+                            // Warning
+                        }
```

Comment:
I understand why you want to add these here eventually, but are they needed right now?